.specify/memory/constitution.md

```diff
@@ -0,0 +1,139 @@
+<!--
+Sync Impact Report:
+Version Change: N/A → 1.0.0
+Initial Constitution Creation
```

Comment:
the objective of this content is unclear. perhaps this would be useful in a changelog but certainly seems wasteful to include it in the document that is the core instructions of all interactions with agents.